apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -5,15 +5,22 @@ module.exports = {
     "airbnb",
     "prettier",
     "plugin:json/recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
We can add the recommended rules for the import plugin: https://github.com/import-js/eslint-plugin-import#typescript
```suggestion
    "plugin:@typescript-eslint/recommended",
    "plugin:import/recommended",
    "plugin:import/typescript",
```

We can probably also add the typed rules for typescript: https://typescript-eslint.io/docs/linting/typed-linting/